pinot-query-planner/src/main/java/org/apache/pinot/query/planner/nodes/AbstractStageNode.java

```diff
@@ -20,18 +20,25 @@
 
```

Comment:
(nit) shall we rename this package to `pinot.query.planner.nodes.logical` or just `pinot.query.planner.logical`